ui/src/pages/AllStories/index.tsx

```diff
@@ -232,11 +236,40 @@ export const AllStories: React.FC = () => {
     dispatch({ type: "SET_TAB_VALUE", newValue });
   };
 
+  const toast = React.useRef(null);
+
+  const showToast = (message: string) => {
+    console.log("TRYING TO OPEN TOAST");
+    toast.current.showToast(message);
+  }
+
+  const publishMap = () => {
+
+    console.log("WE ARE HERE RN");
+
+    if (state.changedVisibility.length > 0) {
```

Comment:
I think we can remove this condition because you are passing in publishMapDisabled so publish button would be disabled if length is 0